doc.md

```diff
@@ -823,6 +824,14 @@ Format string of the file time shown in the info column when it matches this yea
 
 Format string of the file time shown in the info column when it doesn't match this year.
 
+## locale (string) (default `''`)
+
+An IETF BCP 47 language tag for specifying locale used when when using sort type
+`natural` and `name`.
+
+Empty string means disable locale ordering, and a special value `'*'` is used
+to indicate reading locale setting from system environment.
```

Comment:
For me personally, I have been setting my system locale set to `en-US` for a long time, so that system service won't automatically create user directories in Chinese, which are harder to type for `cd` command.

And recently, to enable Dolphin's locale sorting (a GUI file manager if you've never herd about it), I change my system locale to `zh-CN`, and I found this quite annoying.

Reply:
OK I had not considered the situation where a user might have media in other languages (e.g. Japanese anime being a common one). In this case I think it's probably fine to keep it as is (have both `set locale` and `setlocal locale`).

I just wasn't sure at the time whether you were just copying the related code, or whether you actually wanted this. Thanks for confirming.